index.html

```diff
@@ -1642,17 +1642,12 @@ <h4>DIDCommMessaging</h4>
       "https://www.w3.org/ns/did/v1",
       "https://didcomm.org/messaging/contexts/v2"
```

Comment:
Not exactly related to this PR, but I think it's worth noting that this URL returns 404. It seems this issue was addressed some time ago (see https://github.com/decentralized-identity/didcomm.org/issues/9) but somehow the link is broken again.

BTW the context file located at https://github.com/decentralized-identity/didcomm.org/blob/main/docs/messaging/contexts/v2/index.json seems to be using the old format.